Makefile

```diff
@@ -220,8 +220,10 @@ LDFLAGS ?=
 CGO_CFLAGS = $(filter-out -g%,$(CFLAGS))
 CGO_CXXFLAGS = $(CXXFLAGS)
 CGO_LDFLAGS = $(filter-out -static,$(LDFLAGS))
+# certain time based fail if UTC isn't the default timezone
```

Comment:
nit: `Certain time based operations fail`

Can you describe what this approach is? I'm always hesitant to add a new global. Might be worth consulting dev-inf.

Reply:
We have lots of timestamp tests that rely on the local timezone being UTC.   This was achieved by writing to the time.Local to force it to be UTC.   I changed it to achieve that via an env var instead.   Here are the relevant Go docs: https://golang.org/src/time/zoneinfo.go?#L78

Reply:
ah its just an env var. this seems reasonable to me.

Reply:
Cool, who can I ask about how to do this for bazel?

Reply:
dev-inf